soil_api/models/soil_property.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List
+
+from pydantic import BaseModel, Field
+
+from soil_api.models.shared import FeatureType, PointGeometry
+
+
+class SoilPropertiesNames(Enum):
+    bdod = "Bulk density"
+    cec = "Cation exchange capacity (CEC pH 7)"
+    cfvo = "Coarse fragments"
+    clay = "Clay"
+    nitrogen = "Nitrogen"
+    ocd = "Organic carbon density"
+    ocs = "Organic carbon stocks"
+    phh2o = "pH water"
+    sand = "Sand"
+    silt = "Silt"
+    soc = "Soil organic carbon"
+
+
+class SoilPropertiesCodes(Enum):
+    bdod = "bdod"
+    cec = "cec"
+    cfvo = "cfvo"
+    clay = "clay"
+    nitrogen = "nitrogen"
+    ocd = "ocd"
+    ocs = "ocs"
+    phh2o = "phh2o"
+    sand = "sand"
+    silt = "silt"
+    soc = "soc"
+
+
+class SoilPropertiesUnits(Enum):
+    bdod = "cg/cm³"
+    cec = "mmol(c)/kg"
+    cfvo = "cm³/dm³"
+    clay = "g/kg"
+    nitrogen = "cg/kg"
+    ocd = "hg/m³"
+    ocs = "t/ha"
+    phh2o = "pH*10"
+    sand = "g/kg"
+    silt = "g/kg"
+    soc = "dg/kg"
+
+
+class SoilProperty(BaseModel):
+    value: int | str = Field(
+        ..., description="The value of the queried soil property", example=50
+    )
+    unit: str = Field(
```

Comment:
Shouldn't this be a `SoilPropertiesUnits`?

Reply:
Again, this class isn't used